Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -33,9 +33,8 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
          * @param {Object} e ObjectRelations event facade
          */
         _loadObjectRelations: function (e) {
-            var Content = this.get('contentModelConstructor'),
-                loadOptions = {api: this.get('host').get('capi')},
-                relatedContentListArray = [],
+            var that = this,
```

Comment:
nitpick: good resolution idea for the new year, stop writing the ugly `that = this` and bind the function or better here use the third argument of Y.Array.each/Y.Object.each :)